tools/install-source/InstallSourcesTests/MonoPathManglerTest.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using NUnit.Framework;
+
+using InstallSources;
+
+namespace InstallSourcesTests
+{
+	[TestFixture]
+	public class MonoPathManglerTest
+	{
+		MonoPathMangler mangler;
+		string monoPath;
+		string installDir;
+
+		[SetUp]
+		public void SetUp()
+		{
+			monoPath = "/Users/test/xamarin-macios/external/mono/";
```

Comment:
Those are test paths, that is, we are just testing with a random user "test" and this path is passed to the tool in the makefile using the ones defined by make